CognitiveVRTest/Plugins/CognitiveVR/Source/CognitiveVR/Private/CognitiveVR.cpp

```diff
@@ -992,7 +992,7 @@ bool FAnalyticsProviderCognitiveVR::HasEyeTrackingSDK()
 
 bool FAnalyticsProviderCognitiveVR::TryGetRoomSize(FVector& roomsize)
 {
-#if INCLUDE_OCULUS_PLUGIN
+#ifdef INCLUDE_OCULUS_PLUGIN
```

Comment:
Why the change from `#if` to `#ifdef`? Mostly asking out of curiosity.

Reply:
#ifdef checks if it has been defined, which is what we want to check. It causes an error if left as just #if